src/mediatype-icon.ts

```diff
@@ -74,6 +74,11 @@ export class MediatypeIcon extends LitElement {
         display: none;
       }
 
+      svg {
+        height: var(--iconHeight);
+        width: var(--iconWidth);
```

Comment:
What happens when there are no values set for this?

Reply:
it uses the default size it is defined from the svg or the component